app/views/ops/_selected_by_roles.html.haml

```diff
@@ -1,85 +1,8 @@
 -# tree type is servers by role
 - if rec.class == ServerRole
-  .form-horizontal
-    .form-group
-      %label.col-md-2.control-label= _("Role")
-      .col-md-10
-        %p.form-control-static= rec.description
-    .form-group
-      %label.col-md-2.control-label= _("Status")
-      .col-md-10
-        %p.form-control-static= h(@rec_status)
-    .form-group
-      - max = rec.max_concurrent
-      %label.col-md-2.control-label= _("Max Concurrent")
-      .col-md-10
-        %p.form-control-static= h(max == 0 ? _("unlimited") : max)
+  = select_server_by_role_details(rec, @rec_status)
```

Comment:
if these checks can be moved to the helper file, we can replace the `haml` extension to `erb`.